lib/form.js

```diff
@@ -96,29 +121,41 @@ _.extend(Form.prototype, {
     locals: function (/*req, res*/) {
         return {};
     },
+    _checkEmpty: function (req, res, callback) {
```

Comment:
Have gone for checkStatus. It's checking if complete but in theory could be another status. Let me know if you can think of anything better.